test/torch/tensors/test_precision.py

```diff
@@ -449,6 +449,7 @@ def test_torch_sigmoid_approx(workers):
     alice, bob, james = workers["alice"], workers["bob"], workers["james"]
 
     fix_prec_tolerance_by_method = {
+        "chebyshev": {3: 6 / 100, 4: 1 / 100, 5: 1 / 100},
```

Comment:
To improve test speed could we remove the case precision=5 everywhere where it occurs? From what I see =4 is already compelling enough :)